### PR TITLE
docs: Added Apache Spark integration docs

### PR DESCRIPTION
**Pull request by Anush008**
This PR intends to document the [qdrant-spark](https://github.com/qdrant/qdrant-spark) connector on the integrations section.

Section preview: https://deploy-preview-389--condescending-goldwasser-91acf0.netlify.app/documentation/integrations/spark/

### CODE REVIEW

**Comment by Anush008** [comment]
@kacperlukawski, review please.

File: qdrant-landing/content/documentation/integrations/spark.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+```bash
+mvn package -P assembly
+```
+
+This command will compile the source code and generate a fat JAR, which will be stored in the `target` directory by default.
+
+### Maven Central
+
+For Java and Scala projects, you can also obtain the Qdrant-Spark Connector from [Maven Central](https://central.sonatype.com/artifact/io.qdrant/spark).
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/integrations/spark.md:30]: Can we add a dependency snippet for the POM here?

```xml
<dependency>
    <groupId>io.qdrant</groupId>
    <artifactId>spark</artifactId>
    <version>1.6</version>
</dependency>
```

**Review by kacperlukawski — CHANGES_REQUESTED** [review]
Looking good - only one minor change! By the way, will the collection be created if it doesn't exist?

File: qdrant-landing/content/documentation/integrations/spark.md (hunk 1 above)
Comment (Anush008) [reply qdrant-landing/content/documentation/integrations/spark.md:29]: The collection has to be created in advance. I've emphasized the notice regarding this.
https://github.com/qdrant/landing_page/pull/389/commits/b468621c2f36308dc5fa13ba56e3dd4fb29fa282

**Review by kacperlukawski — APPROVED** [review]

**Pull request merged**